docker-compose.yml

```diff
@@ -27,4 +27,7 @@ services:
     cap_add:
       - NET_ADMIN
       - SYS_PTRACE
-
+    healthcheck:
+      test: "ss -l | grep -P 'LISTEN.+:smtp' || exit 1"
```

Comment:
``` SUGGESTION
      test: "ss --listening | grep -P 'LISTEN.+:smtp' || exit 1"
```

Reply:
> which makes it clear for everyone which flags are used and why.

The same would apply for `grep -P` --> `grep --perl-regexp` I guess? 😉